ukaea/fair-mast
# Update metadata index in intake catalogs.

This updates the intake catalog descriptions to match how we are now structuring the s3 storage and adding a future option to include different "levels" of data.

To test:
 - Run a local server and go to http://localhost:8081/load_data.html and check it looks ok/reads ok
 - Run the load_data.ipynb notebook example.
